spec/features/content_item_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.feature "Content Item Details", type: :feature do
```

Comment:
Well done on the feature test.  Two comments:

1. Could you please add this test to the first commit? No need to fail, you can mark it as pending
2. I would change the expectation test. It is better not to test the current path, but something in the view that implies that it is currently working fine. For example, the content item title.